### ISSUE TITLE
No more working?

### ISSUE THREAD

**Issue by alamboley** [reyco1/as3mul#6]
Hey there,

I'm probably missing something dumb, but I can't success to run the `MultiCursorExample`. I've added a Cirrus key, but two computers can't detect each other. Note that the connection to the network doesn't work at all.

Also if I compile targetting the FlashPlayer I get `ArgumentError: Error #2004: One of the parameters is invalid`. Does AS3Mul can only worket with AIR?

Finally is there any way to remove Cirrus usage? I just want to make a P2P on a LAN. I try removing the key etc. and puting "rtmfp:" but without success. I can connect but computers can't detect each other.

**Comment by YopSolo** [comment]
did you tried "rtmfp:" ?
you dn't need Cirrus for p2p on a LAN

**Comment by alamboley** [comment]
Yes absolutely, that's my third part message ;) I forgot to mention that works fine using the `DirectLanConnection.as`class which involved "rtmfp:" without Cirrus. However I would need the whole AS3Mul lib for not reinventing the wheel ;)
